vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -135,13 +135,10 @@ def pre_build(self):
         # If targets aren't explicitly provided then try to target all figures on the page. In this case we don't
         # want to raise an error if the column is not found in a figure's data_frame, it will just be ignored.
         # This is the case when bool(self.targets) is False.
-        # Possibly in future this will change (which would be breaking change).
-        proposed_targets = self.targets or [
-            model.id
-            for model in cast(
-                Iterable[FigureType], model_manager._get_models(FIGURE_MODELS, model_manager._get_model_page(self))
-            )
-        ]
+        # If filter used within container and if targets aren't explicitly provided it will target all figures within
+        # that container. Possibly in future this will change (which would be breaking change).
```

Comment:
What could lead here to the breaking change? 
```suggestion
        # If filter used within container and if targets aren't explicitly provided it will target all figures within
        # that container.
```